README.md

```diff
@@ -1,50 +1,104 @@
+Other businesses may want more custom fields, other companies may want fewer.  There may be between 0 to N custom fields per business.
 
-#### Current State
-![Data Model](images/data_model.png) ![Sample Data](images/sample_data.png)
+## The Challenge
 
-#### Extra Credit
+Using the included API server and your code wizardry, the goal is to implement functionality which allows custom data to be associated with a business' travel employees that are stored in the database.
 
-- Handle invalid custom field values (for example, attempting to set a plain text value to a custom field which accepts a multiple choice value.)
-- Some teams like to specify a custom field on a per-employee basis instead of a per-transaction basis (for example, each employee may have a _Department_ or _Employee ID_). Add an employee model, and allow custom fields to be associated with either employees or transactions.
-  - On `GET /transactions` request, return the value of the custom fields associated with the employee who made the purchase (as well as the values of custom fields associated directly with the transaction).
-- Modify your custom field schema to support multi-select multiple choice. (For example, a user should be able to specify multiple _Cost Centers_ to a single transaction.)
-- Add an endpoint for updating a transaction with new custom data.
-- Consider the performance of your solution when the transactions database begins to contain millions of rows. Ensure that your solution scales elegantly to prevent increased latency on the transactions endpoint with time.
-- There's no security, so anyone can use your new app! For example, one team can currently edit or delete another team's data (BAD!). Try to implement some sort of authentication or authorization.
-- The `GET /transactions` request is going to get quite large as teams continue to use this project. Add a paremeter to this request that allows the user to request specific date ranges, as well as request the report year-to-date, month-to-date, and week-to-date.
+The goal of this task is to build an understanding of how you would structure databases and APIs, so no coding is actually necessary. We will use a whimsical doc and `NOTES.md` to write out your thoughts.
 
-### Installation
+### Before You Get Started
 
-- `$ createdb pana_backend_challenge`
-- `$ cp .env.sample .env`
-- `$ npm install`
-- `$ npm run db:init`
-- `$ npm start`
+We recognize that doing anything within a tight time constraint is an extra level of challenge that you won't face in your day to day job as an engineer.  The goal of this challenge isn't for you to complete everything but for us to get a sense of how you solve problems, so if you don't end up finishing the "core requirements" in the time allotted that is not immediate cause for concern.  Please ask any clarifying questions and work through your thought process as visibly as possible.  We're excited to see what you create!
```

Comment:
I like calling out the fact that the interview may be successful without finishing everything as well as the request for asking clarifying questions and thinking out loud. 👍🏼